cpu/stm32/stmclk/stmclk_l4wb.c

```diff
@@ -124,9 +195,9 @@ void stmclk_init_sysclk(void)
     stmclk_enable_hsi();
 
     /* use HSI as system clock while we do any further configuration and
-     * configure the AHB and APB clock dividers as configure by the board */
-    RCC->CFGR = (RCC_CFGR_SW_HSI | CLOCK_AHB_DIV |
-                 CLOCK_APB1_DIV | CLOCK_APB2_DIV);
+     * configure the AHB and APB clock dividers as configured by the board */
+    RCC->CFGR = (RCC_CFGR_SW_HSI | CLOCK_AHB_DIV | CLOCK_APB1_DIV | CLOCK_APB2_DIV);
+
 #if defined(CPU_FAM_STM32WB)
```

Comment:
Can `IS_ACTIVE`be used here?

Reply:
same as https://github.com/RIOT-OS/RIOT/pull/14866#discussion_r484874097